cves/kernel/CVE-2013-5634.yml

```diff
@@ -171,8 +179,9 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    I don't believe it could be found as it is a resource management issue.
```

Comment:
Likely not a big deal but for clarity might be worth specifying be found by a fuzzer/ automated tool

Reply:
Fair point, I'll update this. Thanks for the feedback.